lib/account/pages/help_page.dart

```diff
@@ -84,9 +84,12 @@ class _FAQCardState extends State<FAQCard> {
                         const Divider(
                           thickness: 1,
                         ),
-                        Text(
-                          widget.answer,
-                          textAlign: TextAlign.start,
+                        Semantics(
+                          liveRegion: true,
```

Comment:
what does the liveRegion do here? When will this text ever change?

Reply:
The text doesn't change, but this makes the Screen Reader read the text automatically when the card is expanded